netmiko/base_connection.py

```diff
@@ -1961,6 +1971,7 @@ def config_mode(
         """
         output = ""
         if not self.check_config_mode():
+            self.write_channel(self.RETURN)
```

Comment:
I'm proposing this change as an improvement. When I run test the first command never has the prompt in front of it. This is because the it has been read by the check_config_mode(). We see the same behavior in the cisco_ios example where the "configure terminal" has no prompt.

`$ python config_change.py 
Password: 

configure terminal
Enter configuration commands, one per line.  End with CNTL/Z.
cisco1(config)#logging buffered 100000
cisco1(config)#end
cisco1#write mem
Building configuration...
[OK]
cisco1#
`